interp/type.go

```diff
@@ -1282,10 +1294,13 @@ func (t *itype) lookupBinField(name string) (s reflect.StructField, index []int,
 	if !isStruct(t) {
 		return
 	}
-	rt := t.rtype
+	rt := t.TypeOf()
 	if t.cat == valueT && rt.Kind() == reflect.Ptr {
 		rt = rt.Elem()
 	}
+	if rt.Kind() != reflect.Struct {
+		return
+	}
```

Comment:
But if we have a pointer of a pointer, then `rt.Elem()` L1299 will return a reflect.Ptr, and the check L1301 will fail, no? Don't we want some kind of recursion here?
Or maybe I'm confused and this can never happen thanks to L1291..., I'm not sure.
Feel free to disregard that comment if it does not make any sense.

Reply:
ok